mipengine/tests/node/test_views.py

```diff
@@ -59,34 +61,43 @@ def test_sql_injection_get_view_schema():
 def test_sql_injection_create_view_context_id():
     with pytest.raises(ValueError):
         context_id = "drop table data;"
-        columns = ["subjectcode", "dataset", "subjectvisitid", "subjectvisitdate"]
+        columns = ["dataset", "age_value", "gcs_motor_response_scale", "pupil_reactivity_right_eye_result"]
         datasets = ["edsd"]
-        create_view.delay(context_id, str(pymonetdb.uuid.uuid1()).replace("-", ""), json.dumps(columns),
-                          json.dumps(datasets)).get()
+        pathology = "tbi"
+        create_view.delay(context_id, str(pymonetdb.uuid.uuid1()).replace("-", ""), pathology, json.dumps(datasets), json.dumps(columns), "").get()
 
 
 def test_sql_injection_create_view_columns():
     with pytest.raises(ValueError):
         context_id = "regrEssion"
-        columns = ["drop table data;", "dataset", "subjectvisitid", "subjectvisitdate"]
+        columns = ["drop table data;", "age_value", "gcs_motor_response_scale", "pupil_reactivity_right_eye_result"]
         datasets = ["edsd"]
-        create_view.delay(context_id, str(pymonetdb.uuid.uuid1()).replace("-", ""), json.dumps(columns),
-                          json.dumps(datasets)).get()
+        pathology = "tbi"
+        create_view.delay(context_id, str(pymonetdb.uuid.uuid1()).replace("-", ""), pathology, json.dumps(datasets), json.dumps(columns), "").get()
 
 
 def test_sql_injection_create_view_datasets():
     with pytest.raises(ValueError):
         context_id = "regrEssion"
-        columns = ["subjectcode", "dataset", "subjectvisitid", "subjectvisitdate"]
+        columns = ["dataset", "age_value", "gcs_motor_response_scale", "pupil_reactivity_right_eye_result"]
         datasets = ["drop table data;"]
-        create_view.delay(context_id, str(pymonetdb.uuid.uuid1()).replace("-", ""), json.dumps(columns),
-                          json.dumps(datasets)).get()
+        pathology = "tbi"
+        create_view.delay(context_id, str(pymonetdb.uuid.uuid1()).replace("-", ""), pathology, json.dumps(datasets), json.dumps(columns), "").get()
 
```

Comment:
in function/methods with more than one argument call it with keyword arguments
     `create_view.delay(context_id=context_id,  ..etc...  ).get() `